src/app/pages/datasets/components/dataset-form/dataset-form.component.ts

```diff
@@ -190,6 +188,11 @@ export class DatasetFormComponent implements OnInit {
             ? this.translate.instant('Switched to new dataset «{name}».', { name: getDatasetLabel(createdDataset) })
```

Comment:
`Switched to new dataset {name}` may not be always appropriate now.
Maybe let's move this notification to `handleSlideInClosed` in `DatasetDetailsPanelComponent` (or move redirection there).